httpserver/src/main/kotlin/tbdex/sdk/httpserver/models/ExchangesApi.kt

```diff
@@ -1,65 +1,70 @@
 package tbdex.sdk.httpserver.models
 
-import tbdex.sdk.protocol.models.*
+import tbdex.sdk.protocol.models.Close
+import tbdex.sdk.protocol.models.Message
+import tbdex.sdk.protocol.models.Order
+import tbdex.sdk.protocol.models.OrderStatus
+import tbdex.sdk.protocol.models.Quote
+import tbdex.sdk.protocol.models.Rfq
 
 /**
  * Interface representing an API for interacting with TBDex exchanges and related entities.
  */
 interface ExchangesApi {
 
   /**
-   * Retrieves the exchange(s) with the specified ID(s).
+   * Retrieves the exchange with the specified ID.
    *
-   * @param id A list of exchange IDs to retrieve. If null, returns null.
-   * @return A list of [MessageKind] representing exchanges with the specified ID(s), or null if not found.
+   * @param id The ID of the exchange to retrieve.
+   * @return A list of [Message] representing the exchange with the specified ID.
    */
-  fun getExchange(id: List<String>? = null): List<MessageKind>?
+  fun getExchange(id: String): List<Message>
```

Comment:
Documentation: Based on how you're using `getExchange` in `SubmitRfq.kt` and other files, `getExchange` should throw if no matching exchange is found. Not sure if this can be specified in kotlin interfaces, but should at least be mentioned in comment.